java/src/org/openqa/selenium/remote/http/jdk/JdkHttpClient.java

```diff
@@ -129,10 +134,7 @@ protected PasswordAuthentication getPasswordAuthentication() {
           new ProxySelector() {
             @Override
             public List<Proxy> select(URI uri) {
-              if (proxy == null) {
-                return List.of();
-              }
-              if (uri.getScheme().toLowerCase().startsWith("http")) {
+                if (uri.getScheme().toLowerCase().startsWith("http")) {
```

Comment:
broken indentation

Reply:
Run `./scripts/formt.sh` to fix all formatting. RBE build will fail without that.